### DISCUSSION TITLE
Not a fan of the "New to GitHub" category name

### DISCUSSION THREAD

**Discussion by Andre601** [community/community#28264] [Other Feature Feedback, Questions, & Ideas] [answered]
I'm not a big fan of the "New to GitHub" category name.

Given the context from [GitHub Community Discourse ➡️ Discussions Categories Update](https://github.com/orgs/community/discussions/21279) was this category called "GitHub Help" within Discourse.
This name does IMO sound a lot more reasonable, especially given its purpose which is to cover questions regarding GitHub and some of its features/products not covered by a more detailed category.

Reading "New to GitHub" gives me (And probably many others) the impression that this category is **only** for new people asking questions regarding GitHub and not being a place for more general questions about GitHub related stuff.
Granted, "General" exists too, but that category is for me more about feedback rather than actual search for answers, considering that this was literally its purpose before this migration from Discourse.

I hope the name can be reconsidered as, again, I and probably others see it as a category exclusive for new users who don't know GitHub well enough yet.

Alternatively, consider to finally add category descriptions that would be displayed when browsing one. That way could the category's purpose be made much more clear.

**Comment by radar-blip** [comment]
You mean its not for new members? 😕

**Reply by Andre601** [reply]
Quote from the discussion I linked:

> **GitHub Help ➡️ New to GitHub:** Ask questions about your GitHub account and the core features of GitHub
> 
> - How to use Git and GitHub
> - Account questions
> - GitHub Pages

**Comment by jfullstackdev** [comment]
I also requested to remove GitHub Pages under that category because that's not basic, as for me, we are solving hard problems for that

I think it should be GitHub Help too, so as not to discourage others to post there, because others are quite not comfortable seeing that category and maybe ashamed just because of that 

let's ping @queenofcorgis, this might really be a necessary request

**Comment by queenofcorgis** [comment, marked as answer]
Thanks for the feedback and for sharing your perspective! When designing our new categories we wanted to be intentional about the spaces we created. We realize there was (and will continue to be) a beginner population in GitHub and wanted to give them a space where they could ask questions freely and safely. Of course, this category is not just limited to new users, rather, we welcome any questions about the core components of GitHub. Your feedback will be passed along to the team as we want to make sure the Community accurately reflects our users.
